### PR TITLE
Visit language

### PR DESCRIPTION
**Pull request by adambbolduc**
The scenario now contains the language in which to process, if not provided, the language is chosen randomly.

### CODE REVIEW

File: scenariolib/scenario.go [hunk 1]
```diff
@@ -8,7 +8,8 @@ type Scenario struct {
 	Name      string      `json:"name"`
 	Weight    int         `json:"weight"`
 	UserAgent string      `json:"useragent,omitempty"`
-	Events    []JSONEvent `json:"events"`
```
Comment (erocheleau) [inline scenariolib/scenario.go]: Why can you have a scenario without events?

File: scenariolib/visit.go [hunk 2]
```diff
@@ -170,7 +175,8 @@ func (v *Visit) sendSearchEvent(q, actionCause, actionType string, customData ma
 		}
 	}
 
-	if v.Config.AllowEntitlements { // Custom shit for besttech, I don't like it
```
Comment (erocheleau) [inline scenariolib/visit.go]: bring back one liner comment please :(

Comment (adambbolduc) [inline scenariolib/visit.go]: I changed formating in Intellij so that I now use go fmt, this will never happen again.

Comment (erocheleau) [inline scenariolib/visit.go]: haha don't get that beaten over this detail :D

File: scenariolib/config.go [hunk 3]
```diff
@@ -25,8 +25,8 @@ import (
 type Config struct {
 	ScenarioMap           []*Scenario
 	OrgName               string              `json:"orgName"`
-	GoodQueries           []string 		  `json:"randomGoodQueries"`
-	BadQueries            []string 		  `json:"randomBadQueries"`
+	GoodQueries           []string            `json:"randomGoodQueries"`
+	BadQueries            []string            `json:"randomBadQueries"`
```
Comment (erocheleau) [inline scenariolib/config.go]: it sucks that it does those kind of diffs that are not actual diffs but oh well

File: scenariolib/config.go [hunk 4]
```diff
@@ -86,7 +86,7 @@ func (c *Config) RandomQuery(good bool) (string, error) {
 }
 
 // Returns a random query in a specified language
-func (c *Config) RandomQueryInLanguage(good bool, language string) (string, error){
+func (c *Config) RandomQueryInLanguage(good bool, language string) (string, error) {
```
Comment (erocheleau) [inline scenariolib/config.go]: This one I like :D

File: scenariolib/event_search.go [hunk 5]
```diff
@@ -12,7 +12,7 @@ import (
 
 // SearchEvent a struct representing a search, is defined by a query to execute
 type SearchEvent struct {
-	query       string
+	query string
```
Comment (erocheleau) [inline scenariolib/event_search.go]: Go fmt doesn't allign stuff when there are comments in between

File: scenariolib/scenario.go (hunk 1 above)
Comment (erocheleau) [inline scenariolib/scenario.go]: Why? @adambbolduc

File: scenariolib/visit.go [hunk 6]
```diff
@@ -263,7 +269,8 @@ func (v *Visit) sendCustomEvent(actionCause, actionType string, customData map[s
...
+	if v.Config.AllowEntitlements {
+		// Custom shit for besttech, I don't like it
```
Comment (erocheleau) [inline scenariolib/visit.go]: Wait it's still like that? Maybe go fmt doesn't allow it?

**Comment by erocheleau** [comment]
SHIP IT!

**Pull request merged**